src/test/java/org/apache/sysds/test/component/compress/lib/CLALibBinaryCellOpTest.java

```diff
@@ -62,7 +62,7 @@ public class CLALibBinaryCellOpTest {
 		// (LessThanEquals.getLessThanEqualsFnObject()), //
 		// (GreaterThan.getGreaterThanFnObject()), //
 		// (GreaterThanEquals.getGreaterThanEqualsFnObject()), //
-		// (Multiply.getMultiplyFnObject()), //
+		(Multiply.getMultiplyFnObject()), //
```

Comment:
i think i missed this in some commit i made because i was debugging, we should enable all of these again.